manifests/c/ctrl-f/userdiag/24.1.17/ctrl-f.userdiag.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.5-1-19041-3930.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ctrl-f.userdiag
+PackageVersion: 24.1.17
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://userdiag.com/download?winget
+  InstallerSha256: A24E199305150F3CC981F5E23FFAE8840C21A9105D956CBF61B1AE6A50A2BBB8
```

Try like that again, i guess it should work
```suggestion
  InstallerSha256: 8E84A8E58670EFECCBEE9A11589A0E1ECD28DB2420F367FF44AAB282F52BCF0A
```